static/js/settings_org.js

```diff
@@ -688,6 +697,17 @@ function _set_up() {
                 opts.data.disallow_disposable_email_addresses = false;
                 opts.data.restricted_to_domain = false;
             }
+
+            var user_invite_restriction = $('#id_realm_user_invite_restriction').val();
+            if (user_invite_restriction === 'no_invite_required') {
+                opts.data.invite_required = false;
```

Comment:
I think we want this to set `invite_by_admins_only = false` as well, right?  Just from a general cleanlyness perspective.